src/codemagic/__version__.py

```diff
@@ -1,5 +1,5 @@
 __title__ = 'codemagic-cli-tools'
 __description__ = 'CLI tools used in Codemagic builds'
-__version__ = '0.29.1.dev'
+__version__ = '0.29.2.dev'
```

Comment:
I am not sure if we need to keep .dev suffix here

Reply:
Yes, `.dev` is fine here. Final release version will come from version declared in [`pyproject.toml`](https://github.com/codemagic-ci-cd/cli-tools/blob/fix/match-mac-app-direct-profile/pyproject.toml#L3), which is currently not updated.